gateway/queue/types.go

```diff
@@ -11,6 +11,7 @@ type Request struct {
 	Header      http.Header
 	Body        []byte
 	Method      string
+	Path        string
```

Comment:
👍 do we have the equivalent change lined-up in nats-queue-worker @telackey @ivanayov ?

Reply:
No, I am not in a good position to test async functionality.

Reply:
> I am not in a good position to test async functionality

What am I missing? Testing the async call is as simple as deploying OpenFaaS on a new VM, deploying your gateway and then invoking the function. Check the logs on the queue-worker and you're good.

Reply:
Surely the question I was responding to about "equivalent change lined-up in nats-queue-worker" must also mean changing, building, and deploying the new nats-queue-worker too.